controllers/storagecluster/cephcluster.go

```diff
@@ -343,6 +343,9 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		}
 	}
 
+	// Update the currentMonCount field in StoragCluster status from the cephCluster CR
+	sc.Status.CurrentMonCount = cephCluster.Spec.Mon.Count
```

Comment:
We should update this only for internal mode. For external mode, this would be undefined.

Reply:
Ah, sure. Updated the code.